extensions/sql-database-projects/src/test/testContext.ts

```diff
@@ -21,6 +21,91 @@ export const mockDacFxResult = {
 	report: ''
 };
 
+export const mockDacFxOptionsResult: mssql.DacFxOptionsResult = {
```

Comment:
I vaguely remember... did Charles had some way for common test file between extensions? that way you wont have to define this twice.

Reply:
You can't share files directly currently.

Although at some point I did want to investigate having the extension tsconfigs pull in common files and compile them. I don't know if there's anything that would prevent it but theoretically that should work?

Otherwise your other options are :

1. NPM package
2. Expose stuff through extension API

Between those two #2 is probably the one I'd suggest if you felt this was really worth it. But in general for test stuff I think it's fine to just duplicate the code for now - especially if it's only going to be used by 2 extensions. 

We can have further discussion on this though since it's a common thing we run in to and would be nice to have a solid answer for.